tests/helpers/test_template.py

```diff
@@ -571,17 +572,17 @@ def test_from_json(hass):
 
 def test_min(hass):
     """Test the min filter."""
-    assert template.Template("{{ [1, 2, 3] | min }}", hass).async_render() == "1"
+    assert template.Template("{{ [1, 2, 3] | min }}", hass).async_render() == 1
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6294088)